torch/_torch_docs.py

```diff
@@ -772,6 +780,12 @@ def parse_kwargs(desc):
              -1.0969, -0.4614],
             [-0.1034, -0.5790,  0.1497, -0.1034, -0.5790,  0.1497, -0.1034,
              -0.5790,  0.1497]])
+    >>> x = torch.ones(2, 3)
```

Comment:
how about an additional example where all the tensors don't have the same dimensionality?

Reply:
In any case, all inputs should have the same .dim() but the size of each dimension could be different if pad_value!=None.